deploy/usb_update.sh

```
@@ -0,0 +1,120 @@
+
+# 5. Cleanup
+echo "Unmounting USB..."
+umount "$MOUNT_POINT"
+echo -e "${GREEN}USB stick can be removed now.${NC}"
+
+# 6. Rebuild
+echo -e "\n${YELLOW}Rebuilding Application...${NC}"
+cd "$DEPLOY_DIR"
+
+if [ -f .env ]; then
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)